### PR TITLE
Removes references to the archive installer

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

We discussed removing the archive install as a recommended method.
This pull request (PR) removes the zip_installer topic and all links to it.
The install guide image was updated to remove the ref to the archive.

## Affected DevDocs pages

-  Many

### CODE REVIEW

File: src/_includes/install/file-system-perms-oneuser_24.md [hunk 1]
```diff
@@ -34,13 +34,12 @@ To optionally enter all commands on one line, enter the following assuming Magen
...
 
-1. If you haven't done so already, get the Magento software in one of the following ways:
+1. If you have not done so already, get the Magento software in one of the following ways:
 
-   *  [Compressed archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html)
    *  [Composer metapackage]({{ page.baseurl }}/install-gde/composer.html)
    *  [Clone the repository (contributing developers only)]({{ page.baseurl }}/install-gde/prereq/dev_install.html)
 
-1. After you have set file system ownership and permissions, [install Magento]({{ page.baseurl }}/install-gde/install/cli/install-cli.html)
+2. After you have set file system ownership and permissions, [install Magento]({{ page.baseurl }}/install-gde/install/cli/install-cli.html)
```
Comment (jeff-matthews) [inline src/_includes/install/file-system-perms-oneuser_24.md:42]: FYI - "1" is preferred for all ordered list items so that we don't have to re-number lists when they change.

File: src/_includes/install/flow-diagram-24.md [hunk 2]
```diff
@@ -8,14 +8,12 @@ The diagram shows the following:
...
 1. Get the Magento software.
 
-   *  For simplicity, get a compressed {{site.data.var.ce}} or {{site.data.var.ee}} [archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html), extract it on your Magento server, and start your installation.
-
    *  If you are more technical and you are familiar with Composer, get a {{site.data.var.ce}} or {{site.data.var.ee}}  [metapackage]({{page.baseurl}}/install-gde/composer.html).
 
    *  If you want to contribute to the {{site.data.var.ce}} codebase or customize the Magento application, [clone]({{ page.baseurl }}/install-gde/prereq/dev_install.html) the Magento 2 GitHub repository. (This method requires familiarity with both GitHub and Composer.)
 
-1. Install the Magento software using the command line.
+2. Install the Magento software using the command line.
```
Comment (jeff-matthews) [inline src/_includes/install/flow-diagram-24.md:15]: unnecessary change

File: src/_includes/install/flow-diagram-24.md [hunk 3]
```diff
@@ -8,14 +8,12 @@ The diagram shows the following:
...
 
    If the step fails because prerequisite software isn't set up correctly, review our [Prerequisites]({{ page.baseurl }}/install-gde/prereq/prereq-overview.html).
 
-1. Verify the installation by viewing your storefront and the Magento Admin.
+3. Verify the installation by viewing your storefront and the Magento Admin.
```
Comment (jeff-matthews) [inline src/_includes/install/flow-diagram-24.md:19]: unnecessary change

File: src/_includes/install/flow-diagram.md [hunk 4]
```diff
@@ -8,16 +8,14 @@ The diagram shows the following:
...
 
    *  If you want to contribute to the {{site.data.var.ce}} codebase or customize the Magento application, [clone]({{ page.baseurl }}/install-gde/prereq/dev_install.html) the Magento 2 GitHub repository. (This method requires familiarity with both GitHub and Composer.)
 
    {:.bs-callout-info}
    To be able to use the Web Setup Wizard to install or upgrade the Magento software, or to manage extensions you get from Magento Marketplace, you must either get a compressed archive or a Composer metapackage. If you clone the GitHub repository, you *cannot* use the Web Setup Wizard to upgrade the Magento software and extensions. You must upgrade using [Composer and Git commands]({{ page.baseurl }}/install-gde/install/cli/dev_options.html).
 
-1. Install the Magento software using either the Web Setup Wizard or command line.
+2. Install the Magento software using either the Web Setup Wizard or command line.
```
Comment (jeff-matthews) [inline src/_includes/install/flow-diagram.md:18]: unnecessary change

File: src/_includes/install/flow-diagram.md [hunk 5]
```diff
@@ -27,4 +25,4 @@ The diagram shows the following:
 
    If the step fails because prerequisite software isn't set up correctly, review our [Prerequisites]({{ page.baseurl }}/install-gde/prereq/prereq-overview.html).
 
-1. Verify the installation by viewing your storefront and the Magento Admin.
+3. Verify the installation by viewing your storefront and the Magento Admin.
```
Comment (jeff-matthews) [inline src/_includes/install/flow-diagram.md:28]: unnecessary change

File: src/guides/v2.3/install-gde/prereq/nginx.md [hunk 6]
```diff
@@ -99,19 +97,19 @@ You cannot use the Web Setup Wizard when installing Magento on nginx. You must u
 
 1. As the [Magento file system owner]({{page.baseurl}}/install-gde/prereq/file-sys-perms-over.html), log in to your Magento server.
 
-1. Change to the web server docroot directory or a directory that you have configured as a virtual host docroot. For this example, we're using the Ubuntu default `/var/www/html`.
+2. Change to the web server docroot directory or a directory that you have configured as a virtual host docroot. For this example, we're using the Ubuntu default `/var/www/html`.
```
Comment (jeff-matthews) [inline src/guides/v2.3/install-gde/prereq/nginx.md:100]: unnecessary change

File: src/guides/v2.3/install-gde/prereq/nginx.md [hunk 7]
```diff
@@ -99,19 +97,19 @@ You cannot use the Web Setup Wizard when installing Magento on nginx. You must u
...
 
    ```bash
    cd /var/www/html
    ```
 
-1. Install Composer globally. You'll need Composer to update dependencies before installing Magento:
+3. Install Composer globally. You'll need Composer to update dependencies before installing Magento:
```
Comment (jeff-matthews) [inline src/guides/v2.3/install-gde/prereq/nginx.md:106]: unnecessary change

File: src/guides/v2.3/install-gde/prereq/nginx.md [hunk 8]
```diff
@@ -99,19 +97,19 @@ You cannot use the Web Setup Wizard when installing Magento on nginx. You must u
...
 
    ```bash
    curl -sS https://getcomposer.org/installer | sudo php -- --install-dir=/usr/bin --filename=composer
    ```
 
-1. Create a new Composer project using the {{site.data.var.ce}} or {{site.data.var.ee}} metapackage.
+4. Create a new Composer project using the {{site.data.var.ce}} or {{site.data.var.ee}} metapackage.
```
Comment (jeff-matthews) [inline src/guides/v2.3/install-gde/prereq/nginx.md:112]: unnecessary change

File: src/guides/v2.3/install-gde/prereq/nginx.md [hunk 9]
```diff
@@ -127,7 +125,7 @@ You cannot use the Web Setup Wizard when installing Magento on nginx. You must u
 
    When prompted, enter your [Magento authentication keys]({{ page.baseurl }}/install-gde/prereq/connect-auth.html). Your _public key_ is your username; your _private key_ is your password.
 
-1. Set read-write permissions for the web server group before you install the Magento software. This is necessary so that the Setup Wizard and command line can write files to the Magento file system.
+5. Set read-write permissions for the web server group before you install the Magento software. This is necessary so that the Setup Wizard and command line can write files to the Magento file system.
```
Comment (jeff-matthews) [inline src/guides/v2.3/install-gde/prereq/nginx.md:128]: unnecessary change

File: src/guides/v2.3/install-gde/prereq/nginx.md [hunk 10]
```diff
@@ -149,7 +147,7 @@ You cannot use the Web Setup Wizard when installing Magento on nginx. You must u
    chmod u+x bin/magento
    ```
 
-1. Install Magento from the [command line]({{ page.baseurl }}/install-gde/install/cli/install-cli.html). This example assumes that the Magento install directory is named `magento2ee`, the `db-host` is on the same machine (`localhost`), and that the `db-name`, `db-user`, and `db-password` are all `magento`:
+6. Install Magento from the [command line]({{ page.baseurl }}/install-gde/install/cli/install-cli.html). This example assumes that the Magento install directory is named `magento2ee`, the `db-host` is on the same machine (`localhost`), and that the `db-name`, `db-user`, and `db-password` are all `magento`:
```
Comment (jeff-matthews) [inline src/guides/v2.3/install-gde/prereq/nginx.md:150]: unnecessary change

File: src/guides/v2.3/install-gde/prereq/nginx.md [hunk 11]
```diff
@@ -170,7 +168,7 @@ You cannot use the Web Setup Wizard when installing Magento on nginx. You must u
    --use-rewrites=1
    ```
 
-1. Switch to developer mode:
+7. Switch to developer mode:
```
Comment (jeff-matthews) [inline src/guides/v2.3/install-gde/prereq/nginx.md:171]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 12]
```diff
@@ -88,27 +88,25 @@ There are several ways to download the Magento software, including:
...
-*  [Download an archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html)
-
 *  [Clone the git repository]({{ page.baseurl }}/install-gde/prereq/dev_install.html)
 
 For this example, we'll install using Composer and the command line.
 
 1. As the [Magento file system owner]({{page.baseurl}}/install-gde/prereq/file-sys-perms-over.html), log in to your Magento server.
 
-1. Change to the web server docroot directory or a directory that you have configured as a virtual host docroot. For this example, we're using the Ubuntu default `/var/www/html`.
+2. Change to the web server docroot directory or a directory that you have configured as a virtual host docroot. For this example, we're using the Ubuntu default `/var/www/html`.
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:97]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 13]
```diff
@@ -88,27 +88,25 @@ There are several ways to download the Magento software, including:
...
 
    ```bash
    cd /var/www/html
    ```
 
-1. Install Composer globally. You'll need Composer to update dependencies before installing Magento:
+3. Install Composer globally. You'll need Composer to update dependencies before installing Magento:
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:103]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 14]
```diff
@@ -88,27 +88,25 @@ There are several ways to download the Magento software, including:
...
 
    ```bash
    curl -sS https://getcomposer.org/installer | sudo php -- --install-dir=/usr/bin --filename=composer
    ```
 
-1. Create a new Composer project using the {{site.data.var.ce}} or {{site.data.var.ee}} metapackage.
+4. Create a new Composer project using the {{site.data.var.ce}} or {{site.data.var.ee}} metapackage.
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:109]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 15]
```diff
@@ -124,7 +122,7 @@ For this example, we'll install using Composer and the command line.
 
    When prompted, enter your [Magento authentication keys]({{ page.baseurl }}/install-gde/prereq/connect-auth.html). Your _public key_ is your username; your _private key_ is your password.
 
-1. Set read-write permissions for the web server group before you install the Magento software. This is necessary so that the command line can write files to the Magento file system.
+5. Set read-write permissions for the web server group before you install the Magento software. This is necessary so that the command line can write files to the Magento file system.
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:125]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 16]
```diff
@@ -146,7 +144,7 @@ For this example, we'll install using Composer and the command line.
    chmod u+x bin/magento
    ```
 
-1. Install Magento from the [command line]({{ page.baseurl }}/install-gde/install/cli/install-cli.html). This example assumes that the Magento install directory is named `magento2ee`, the `db-host` is on the same machine (`localhost`), and that the `db-name`, `db-user`, and `db-password` are all `magento`:
+6. Install Magento from the [command line]({{ page.baseurl }}/install-gde/install/cli/install-cli.html). This example assumes that the Magento install directory is named `magento2ee`, the `db-host` is on the same machine (`localhost`), and that the `db-name`, `db-user`, and `db-password` are all `magento`:
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:147]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 17]
```diff
@@ -167,7 +165,7 @@ For this example, we'll install using Composer and the command line.
    --use-rewrites=1
    ```
 
-1. Switch to developer mode:
+7. Switch to developer mode:
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:168]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 18]
```diff
@@ -364,27 +362,25 @@ There are several ways to download the Magento software, including:
...
-*  [Download an archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html)
-
 *  [Clone the git repository]({{ page.baseurl }}/install-gde/prereq/dev_install.html)
 
 For this example, we'll install using Composer and the command line.
 
 1. As the [Magento file system owner]({{page.baseurl}}/install-gde/prereq/file-sys-perms-over.html), log in to your Magento server.
 
-1. Change to the web server docroot directory or a directory that you have configured as a virtual host docroot. For this example, we're using the Ubuntu default `/var/www/html`.
+2. Change to the web server docroot directory or a directory that you have configured as a virtual host docroot. For this example, we're using the Ubuntu default `/var/www/html`.
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:371]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 19]
```diff
@@ -364,27 +362,25 @@ There are several ways to download the Magento software, including:
...
 
    ```bash
    cd /var/www/html
    ```
 
-1. Install Composer globally. You'll need Composer to update dependencies before installing Magento:
+3. Install Composer globally. You'll need Composer to update dependencies before installing Magento:
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:377]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 20]
```diff
@@ -364,27 +362,25 @@ There are several ways to download the Magento software, including:
...
 
    ```bash
    curl -sS https://getcomposer.org/installer | sudo php -- --install-dir=/usr/bin --filename=composer
    ```
 
-1. Create a new Composer project using the {{site.data.var.ce}} or {{site.data.var.ee}} metapackage.
+4. Create a new Composer project using the {{site.data.var.ce}} or {{site.data.var.ee}} metapackage.
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:383]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 21]
```diff
@@ -400,7 +396,7 @@ For this example, we'll install using Composer and the command line.
 
    When prompted, enter your [Magento authentication keys]({{ page.baseurl }}/install-gde/prereq/connect-auth.html). Your _public key_ is your username; your _private key_ is your password.
 
-1. Set read-write permissions for the web server group before you install the Magento software. This is necessary so that the and command line can write files to the Magento file system.
+5. Set read-write permissions for the web server group before you install the Magento software. This is necessary so that the and command line can write files to the Magento file system.
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:399]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 22]
```diff
@@ -422,7 +418,7 @@ For this example, we'll install using Composer and the command line.
    chmod u+x bin/magento
    ```
 
-1. Install Magento from the [command line]({{ page.baseurl }}/install-gde/install/cli/install-cli.html). This example assumes that the Magento install directory is named `magento2ee`, the `db-host` is on the same machine (`localhost`), and that the `db-name`, `db-user`, and `db-password` are all `magento`:
+6. Install Magento from the [command line]({{ page.baseurl }}/install-gde/install/cli/install-cli.html). This example assumes that the Magento install directory is named `magento2ee`, the `db-host` is on the same machine (`localhost`), and that the `db-name`, `db-user`, and `db-password` are all `magento`:
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:421]: unnecessary change

File: src/guides/v2.4/install-gde/prereq/nginx.md [hunk 23]
```diff
@@ -443,7 +439,7 @@ For this example, we'll install using Composer and the command line.
    --use-rewrites=1
    ```
 
-1. Switch to developer mode:
+7. Switch to developer mode:
```
Comment (jeff-matthews) [inline src/guides/v2.4/install-gde/prereq/nginx.md:442]: unnecessary change

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
Please add a redirect from the `src/guides/v2.3/install-gde/prereq/zip_install.md` file to the Composer install topic.

Also, you need to remove the archive install topic from the TOOC @dobooth.

File: src/_includes/install/file-system-perms-oneuser_24.md (hunk 1 above)
Comment (dobooth) [reply src/_includes/install/file-system-perms-oneuser_24.md:42]: Strange. I didn't think I touched those. I love the auto-numbering thing.

**Review by jeff-matthews — APPROVED** [review]

**Comment by dobooth** [comment]
No errors via rake.
